app/models/curation_concerns/_form_additional_information.html.erb

```diff
@@ -0,0 +1,12 @@
+<fieldset class="optional prompt">
+  <legend>Additional Information</legend>
+  <%= f.input :subject,                as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :publisher,              as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :date %>
+  <%= f.input :source,                 as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :language,               as: :multi_value, input_html: { class: 'form-control' } %>
+  <%= f.input :isbn,                   as: :multi_value, input_html: { class: 'form-control' }, label: 'ISBN' %>
```

Comment:
FWIW, I wouldn't call it a story. A story must be something that's visible to the users, this wouldn't be.  This is more "keeping everything organized" so you know what file to open to change it.